examples/step-79/doc/intro.dox

```diff
@@ -0,0 +1,597 @@
+last accepted step is taken to be the new watchdog step, and the process is
+repeated.
+
+
+Above, the ``maximal feasible step" is a scaling of the newton step in both the
+primal and dual variables given by
+
+@f[
+  \beta^\textbf{y} = \min\{1,\max \beta \text{ such that }\left(\textbf{z}_{k+i}
+   + \beta^\textbf{z}_{k+i} \Delta \textbf{z}_{k+i}\right)_j \geq \zeta
+   \textbf{z}_{k+i,j} \forall j\}
+@f]
+@f[
+  \beta^\textbf{x} = \min\{1,\max \beta \text{ such that }\left(\textbf{s}_{k+i}
+   + \beta^\textbf{s}_{k+i} \Delta \textbf{s}_{k+i}\right)_j \geq \zeta
+   \textbf{s}_{k+i,j} \forall j\}
+@f]
+
+Above, $\zeta$ is the "fraction to boundary" that is allowed on any step.
+Because the derivatives become ill-conditioned near the boundary, this technique
+ stands in for a trust region and is necessary to ensure good approximations in
+ the future. $\zeta$ is taken to be $\max\{.8, 1-\alpha\}$, which allows
+ movement closer to the boundary as the barrier becomes smaller. In the future,
+ when implementing the LOQO algorithm for barrier reduction, this must be kept
+ to .8 as the barrier parameter can vary wildly.
+
+Separately, we need to deal with the log-barrier that we have used to enforce
+the positivity constraint on the slack variables $s_1,s_2$: In the statement of
+the final optimization problem we solve, we have added the term
+@f[
+  -\alpha \int_\Omega (\log(s_1) + \log(s_2)).
+@f]
+The question is how we should choose the penalty factor $\alpha$. As with all
+penalty methods, we are in reality only interested in the limit as
+$\alpha\to 0$, since this is then the problem we really wanted to solve,
+subject to the positivity constraints on the slack variables. On the other hand,
+ we need to choose $\alpha$ large enough to make the problem solvable in
+ practice. Actual implementations therefore start with a larger value of
+ $\alpha$ and gradually decrease it as the outer iterations proceed.
+
+In the monotone method implemented here, the barrier parameter is updated
+whenever some level of convergence is achieved at the current barrier parameter.
+ We use the $l_\infty$ norm of the KKT conditions to check for convergence at
+ each barrier size. The requirement is that
+ $\norm{KKT}_{l_\infty} < c \cdot \alpha$ where $c$ is a constant over any
+ barrier size and $\alpha$ is the barrier parameter. This forces better
+ convergence in later iterations, and is the same requirement as is used in
+ IPOPT.
+
+Here, the barrier is reduced linearly at larger values, and superlinearly at
+smaller values. At larger values, it is multiplied by a constant (around .6),
+and at lower values the barrier value is replaced by the barrier value raised
+to some exponent (around 1.2). This method has proven to be effective at keeping
+ the subproblem solvable at large barrier values, while still allowing
+ superlinear convergence at smaller barrier values.
+@f[
+  \alpha_{k+1} = \min\{\alpha_k^{1.2},.6\alpha_k\}
```

Comment:
This equation requires some introduction. Its just "floating" below the paragraph above.